dotenv/src/parse.rs

```diff
@@ -290,7 +290,7 @@ KEY4='fo ur'
 KEY5="fi ve"
 KEY6=s\ ix
 KEY7=
-KEY8=     
+KEY8=
```

Comment:
KEY8 intentionally had whitespace here

Reply:
Sorry, my editor did this automatically. Fixed.